lib/deobfuscate.py

```diff
@@ -46,7 +63,7 @@ def deobfuscate(payload: str, max_depth: int = DEFAULT_MAX_DEPTH) -> str:
 
         if deobfuscated == payload:
             logger.debug("Payload equals deobfuscated, exiting loop")
-            return deobfuscated
+            return parse_base64_payload(deobfuscated)
```

Comment:
Add the `decode_base64` flag to the `deobfuscate` function

Reply:
Fixed